### ISSUE TITLE
question on xmi / json

### ISSUE THREAD

**Issue by jwijffels** [inception-project/inception-external-recommender-v2#23]
I've had a look at this repository as an alternative to https://github.com/inception-project/inception-external-recommender
Main reason is that I would be interested if I could call from inception an external recommender API written in pure R. 
The repository https://github.com/inception-project/inception-external-recommender requires to return a CAS which would require me to write an R wrapper around the UIMA CAS C++ library. I could do that but if on inception-external-recommender-v2 only returning json would do the trick of sending the data back to inception, I don't need to build this R wrapper around the UIMA CAS C++ library.

Is that correct, are you planning to have inception to be able to handle json returned as recommender instead of requiring a cas xmi structure as documented at https://inception-project.github.io/releases/22.3/docs/developer-guide.html#_example_http_response?

**Comment by jcklie** [comment]
Yes. There is also a branch for that in INCEpTION that uses it: https://github.com/inception-project/inception/pull/2313 . I just did not have the time yet to finalize, so things on both sides can change.

**Comment by jwijffels** [comment]
Thanks for the feedback. Looking forward to see that integration :+1:
